spec/game_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative '../lib/game'
+
+describe Game do
+  subject(:game) { described_class.new }
+  it 'plays 10 gutter balls' do
+    20.times { game.roll(0) }
+    expect(game.score).to eq 0
+  end
+
+  it 'keeps track of a score with no bonuses' do
+    20.times { game.roll(2) }
+    expect(game.score).to eq 40
+  end
+
+  it 'adds a bonus for a spare' do
+    game.roll(5); game.roll(5)
+    game.roll(3); game.roll(0)
+    16.times { game.roll(0) }
+    expect(game.score).to eq (5 + 5 + 3) + 3
```

Comment:
really really loving that you lay out the arithmetics of the point counting here - it makes your thought process clear, and it makes it easy to understand for people unfamiliar with the game itself.